Anmol Ahuja/backend/eventHandlers.js

```diff
@@ -0,0 +1,16 @@
+const { addUser } = require('./users')
+
+module.exports = (io) => {
+    function eventHandlers (socket) {
+        console.log('A client just got conencted!!');
+
+        socket.on('user-name', (name) => {
+            io.emit('user-name', name)
+        })
+
+        socket.on('disconenct', () => {

```

Comment:
You might need to change this line, I think you have misspelled it
```suggestion
        socket.on('disconnect', () => {

```
Overall Great work !! 🔥

Reply:
Yup done.
Thanks for the review!